Library/Homebrew/cmd/test-bot.rb

```diff
@@ -583,7 +583,7 @@ def formula(formula_name)
           end
         end
         test "brew", "test", "--verbose", formula_name if formula.test_defined?
-        testable_dependents.each do |dependent|
+        dependents.each do |dependent|
```

Comment:
Unlike the name suggests, `testable_dependents` are not merely dependents with a test, but dependents that have a test and are bottled (see a bit further up). Switching to `dependents` potentially means that unbottled dependents will be built from source, just for the sake of checking their linkage, which is likely to be alright due to the from-source build.